packages/flame/lib/src/gestures/events.dart

```diff
@@ -219,3 +221,7 @@ class ScaleUpdateInfo extends PositionInfo<ScaleUpdateDetails> {
     ScaleUpdateDetails raw,
   ) : super(game, raw.focalPoint, raw);
 }
+
+mixin _HandledField {
```

Comment:
That is cheating :P

Reply:
I thought you said that we'd try out to see how it looked like with a mutable boolean? Or how is it cheating? :smile:

Reply:
I am just messing with you, I liked the approach to create a mixin so you don't need to alter all the base classes

Reply:
why can't this be added only once to `BaseInfo`?